common4j/src/main/com/microsoft/identity/common/java/providers/oauth2/OpenIdProviderConfigurationClient.java

```diff
@@ -113,12 +93,45 @@ public void run() {
      *
      * @return OpenIdProviderConfiguration
      */
-    public synchronized OpenIdProviderConfiguration loadOpenIdProviderConfiguration()
+    public synchronized OpenIdProviderConfiguration loadOpenIdProviderConfigurationFromTenant(@NonNull final String tenantIdentifier)
+            throws ServiceException {
+        try {
+            final String tenantedAuthorityUrl = new CommonURIBuilder()
+                    .setScheme(HTTPS_SCHEME)
+                    .setHost(WELL_KNOWN_CONFIG_HOST)
+                    .setPathSegments(tenantIdentifier)
+                    .build().toString();
```

Comment:
Question about this method. For custom domains (i.e. https://login.subway.com/), can you still get the OpenIdConfiguration by using the WELL_KNOWN_CONFIG_HOST (login.microsoftonline.com)? This is relevant for B2C and eventually CIAM

Reply:
I don't think so, but perhaps we need to figure out the tenant id of that B2C tenant and substitute that in the path portion of the URL - may be then that works.